src/vs/base/common/product.ts

```diff
@@ -288,3 +289,12 @@ export interface IAiGeneratedWorkspaceTrust {
 	readonly dontTrustOption: string;
 	readonly startupTrustRequestLearnMore: string;
 }
+
+export interface IEntitlement {
```

Comment:
This seems GitHub specific. Can you rename this and the property in the product.json